test/e2e/e2e_test.go

```diff
@@ -433,14 +437,15 @@ var _ = Describe("controller", Ordered, func() {
 			EventuallyWithOffset(1, verifyPVCResources, 3*time.Minute, 15*time.Second).Should(Succeed())
 		})
 		It("apply custom rawConfig", func() {
+			Skip("skip raw config test")
 			customConfig := `port 6379
-cluster-enabled yes
-cluster-config-file nodes.conf
-cluster-node-timeout 5000
-appendonly yes
-protected-mode no
-cluster-preferred-endpoint-type ip
-maxmemory 12mb`
+		cluster-enabled yes
```

Comment:
As far as I understand, this indentation will get interpreted causing the valkey conf on disc to look like:

```
port 6379
		cluster-enabled yes
		cluster-config-file nodes.conf
		cluster-node-timeout 5000
		appendonly yes
		protected-mode no
		cluster-preferred-endpoint-type ip
		maxmemory 12mb
```

I'd worry that might not parse as valid given this test doesn't technically assert that the config is valid, just that it gets inserted into a config map.

I should extend this test to also kill the running pod and then assert that the new pod contains the changes, which would sanity check the config here